pkg/kgen/kgen.go

```diff
@@ -24,7 +25,9 @@ func Generate(opts GenerateOpts) *v1.Pod {
 			Kind:       "Pod",
 			APIVersion: "v1",
 		},
-		ObjectMeta: metav1.ObjectMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "default",
+		},
```

Comment:
```suggestion
		ObjectMeta: metav1.ObjectMeta{},
```
I remade a suggestion since you fixed an issue in the meantime :)